aws-athena/ctas/proximity-dist_pin_to_pin_10km.sql

```diff
@@ -0,0 +1,36 @@
+-- CTAS that finds the 3 nearest neighbor PINs for every PIN for every year
+-- within a 10km radius, filtered for PINs that do not have three neighbors
+-- within a 1km radius
+{{
+    config(
+        materialized='table',
+        partitioned_by=['year'],
+        bucketed_by=['pin10'],
+        bucket_count=1
+    )
+}}
+
+WITH missing_matches AS (  -- noqa: ST03
```

Comment:
We ignore [ST03](https://docs.sqlfluff.com/en/stable/rules.html#rule-structure.unused_cte) here because we technically _do_ reference `missing_matches` on line 31 below, but since it's passed into a macro as a string, SQLFluff isn't able to recognize it as a reference. (This context might make sense as an inline comment, but it seemed borderline to me so I left it out; let me know if you disagree!)

Reply:
I think this is fine without being a comment, since it's pretty clear from the code itself.